### PR TITLE
Add Hydra console

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Runs a Generic Hypermedia App container to access the API through.

### CODE REVIEW

File: .docker/docker-compose.yml [hunk 1]
```diff
@@ -6,3 +6,12 @@ services:
       context: ../
     ports:
       - '8080:8080'
+  console:
+    image: tpluscode/hypermedia-app:alpha
+    environment:
+      - 'API_ENTRYPOINTS={ "http://localhost:8080/": "Article Store" }'
+      - 'BASE_URL=http://localhost:8080/'
```
Comment (thewilkybarkid) [inline .docker/docker-compose.yml]: Not working due to https://github.com/hypermedia-app/generic.hypermedia.app/issues/44.

File: .docker/docker-compose.yml [hunk 2]
```diff
@@ -6,3 +6,12 @@ services:
       context: ../
     ports:
       - '8080:8080'
+  console:
+    image: tpluscode/hypermedia-app:alpha
```
Comment (thewilkybarkid) [inline .docker/docker-compose.yml]: Will need a tag released before can merge. /cc @tpluscode

Comment (thewilkybarkid) [reply .docker/docker-compose.yml]: Heh, hadn't noticed https://hub.docker.com/layers/tpluscode/hypermedia-app/version-0.12.0/images/sha256-dc1a9ddf92da97461275fd819a0e5029ae8203027f4e3f1e9205ff5c8b9227b9 was pushed. 🎉

Comment (tpluscode) [reply .docker/docker-compose.yml]: Unfortunately it's still one with the environment variables not honoured (hypermedia-app/generic.hypermedia.app#44). Stay tuned for build 0.12.1

**Comment by tpluscode** [comment]
~~By the way, please note that the `BASE_URL` is a base of the called API and not the client application.~~

~~The container exposes port `8000` (unchanged default of `local-web-server`).~~

never mind

File: .docker/docker-compose.yml (hunk 2 above)
Comment (tpluscode) [reply .docker/docker-compose.yml]: > docker pull tpluscode/hypermedia-app:version-0.12.1

**Review by stephenwf — APPROVED** [review]
Not so keen on unescaped JSON as environment variables, I assume docker does something clever to escape them?
```
- 'API_ENTRYPOINTS={ "http://localhost:8080/": "Article Store" }'
```
Can see from the docs that how it should be configured though.

**Comment by tpluscode** [comment]
> Not so keen on unescaped JSON as environment variables, I assume docker does something clever to escape them?
> 
> ```
> - 'API_ENTRYPOINTS={ "http://localhost:8080/": "Article Store" }'
> ```
> 
> Can see from the docs that how it should be configured though.

While this is a set of key/value pairs, I found JSON to be the simplest to consume. I considered a double-delimited string like `api1=link1;api2=link2`. Let me know if you think that would be better?

**Pull request merged**